wiki/History_of_osu!/Online_rankings/en.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+# Online rankings history
+
+Early versions of osu! were a mostly isolated experience, with no online leaderboards for anything until [October 6, 2007](https://osu.ppy.sh/community/forums/topics/15?n=29), when support for public online rankings was added for [osu!](/wiki/Game_mode/osu!). Since then, all [game modes](/wiki/Game_mode) have recieved [online rankings](https://osu.ppy.sh/rankings/osu/performance).
```

Comment:
```suggestion
Early versions of osu! were a mostly isolated experience, with no online leaderboards for anything until [6 October 2007](https://osu.ppy.sh/community/forums/topics/15?n=29), when support for public online rankings was added for [osu!](/wiki/Game_mode/osu!). Since then, all [game modes](/wiki/Game_mode) have recieved their own [online rankings](https://osu.ppy.sh/rankings/osu/performance).
```